testinfra/test_ami_nix.py

```
@@ -633,6 +633,227 @@ def test_libpq5_version(host):
+        return
+
+    # Check if gatekeeper is installed via Nix
+    result = run_ssh_command(
+        host["ssh"],
+        "sudo -u postgres ls -la /var/lib/postgresql/.nix-profile/lib/security/pam_jit_pg.so 2>/dev/null",
+    )
+    if result["succeeded"]:
+        print(f"\nJIT PAM module found in Nix profile:\n{result['stdout']}")
+    else:
+        print("\nJIT PAM module not found in postgres user's Nix profile")
+        assert False, "JIT PAM module (pam_jit_pg.so) not found in expected location"
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)